src/EditController.js

```diff
@@ -30,16 +34,31 @@ class EditController extends ViewController {
     router.route("POST", routePrefix+"/edit/:id", ::this.save)
     router.route("POST", routePrefix+"/delete/:id", ::this.remove)
 
+    this.redirectAfterCreate = options.redirectAfterCreate || ""
+    this.redirectAfterEdit = options.redirectAfterEdit || ""
+    this.redirectAfterDelete = options.redirectAfterDelete || ""
+    
     // Yes, these should be __dirname not local to the subclass
     // Subclass templates are expected to be loaded by MVCModule or manually?
     templater.default().template(__dirname+"/templates/web-controller-form.ejs", this.pageTemplate, this.templatePrefix+"-create")
     templater.default().template(__dirname+"/templates/web-controller-form.ejs", this.pageTemplate, this.templatePrefix+"-edit")
     templater.default().template(__dirname+"/templates/web-controller-paginator.ejs")
   }
 
+  /*
```

Comment:
My first reaction to this was "why one common function that gets applied to all routes?" (It seemed like an invitation to nasty code.) But then I looked at how you were using it, and felt better.

I offer this up as a possible static helper function for `EditController`, and an example use:
```
  /** Replaces route parameters with values.
   * @param {string} route route path
   * @param {Object} params parameter replacements
   * @returns {string} route path, with parameters replaced
   */
  static replaceRouteParams(route, params) {
    _.each(params, (val, key) => { route = route.replace(":" + key, val) })
    return route
  }

  ...

  routeForRequest(req, route) {
    return EditController.replaceRouteParams(super.routeForRequest(req, route),
      { [this.parentParam]: req.params[this.parentParam] })
  }
```
You're welcome to take it or leave it. (But isn't the new initializer computed property name syntax cool? I hadn't see that!)